docs: Choose analytics tool (#1495)
Flank/flank: docs/analytics_comparsion.md

    ```kotlin
    val messageBuilder = MessageBuilder(PROJECT_TOKEN);
    
    val props = JSONObject()
    props.put("name", args.project) //set name as project id to easiest identification on users list
    val update = messageBuilder.set(project, props)
    
    // Send the update to mixpanel
    val mixpanel = MixpanelAPI()
    mixpanel.sendMessage(update)
    ```
1. Send event

```kotlin

val messageBuilder = MessageBuilder(PROJECT_TOKEN)

// You can send properties along with events
val props =  JSONObject()
props.put("type", "gameloop")
props.put("outputStyle", "single")

val configurationEvent =
    messageBuilder.event(distinctId, "configuration", props)

val api = MixpanelAPI()

api.sendMessage(configurationEvent)
```

You could find working code [here](https://github.com/Flank/flank/tree/1460-mix-panel-poc)

### Results

Configuration and sending events is easy. Mixpanel dashboard is advanced and probably meets our requirements. 
We could send all changed parameters by one event or send changed parameters one by one.

### Pros
- easy to use
- standalone
- easy to configure

## POC Segment

### Scenario
Send configuration provided by a user to Segment.

### Description

> During making POC Segment appears to be a not a standalone solution. It is just a tunnel to map and send data to other destination(s).

Reviewer: ```suggestion
During making POC Segment appears to be not a standalone solution. It is just a tunnel to map and send data to other destination(s).
```

> During making POC Segment appears to be a not a standalone solution. It is just a tunnel to map and send data to other destination(s).

Reviewer: ```suggestion
During making POC Segment appears to be not a standalone solution. It is just a tunnel to map and send data to other destination(s).
```

Segment needs Google Tracking ID for sending events to them, however to have it, [you must
use an old version of Google Analytics(Universal) which is not so easy to find, new Google Analytics does not use this
property](https://support.google.com/analytics/answer/7372977?hl=en) and Segment does not work with the latest version of
Google Analytics.
After configuring Segment and Google Analytics (Universal old version) events are received by the Google tool. However, data are
not readable and could not be analyzed. This tool is mostly used for websites and does not allow querying properties or
custom events. Maybe this is the fault of Segment integration or Flank specific use case, but this tool does not work properly with Google Analytics.
Other destinations are not worth to be a consideration, because it is better to use standalone versions of them.

### Pros
- easy to use

### Cons
- it is not standalone (needs additional tools)
- configuration of destinations is not easy, there is not enough information for it
- implementation force user to use an older version of the end solution (Google Analytics)

## Decision MixPanel vs Segment

> Based on Flank needs it was decided that MixPanel will deliver the best experience to Team. Segment is not standalone tool and its configuration is painful, so it is better to stick to MixPanel which provide easy out of the box integration and met all criteria created by Flank team.

Reviewer: ```suggestion
Based on Flank needs it was decided that MixPanel will deliver the best experience to Team. Segment is not a standalone tool and its configuration is painful, so it is better to stick to MixPanel which provides easy out of the box integration and met all criteria created by Flank team.
```
